src/main/java/com/mixpanel/android/util/HttpService.java

```diff
@@ -107,7 +82,7 @@ public boolean isOnline(Context context) {
                     out = null;
                 }
                 in = connection.getInputStream();
-                response = slurp(in);
+                response = Utilities.slurp(in);
```

Comment:
I'm not sure about this, but do you think it's an overkill of having slurp in the Utilities.java? It's only referenced once in the code base.